onadata/libs/utils/logger_tools.py

```diff
@@ -1,59 +1,48 @@
-from datetime import datetime
 import os
-import pytz
 import re
 import sys
 import tempfile
 import traceback
+from datetime import datetime
+from wsgiref.util import FileWrapper
 from xml.dom import Node
 from xml.parsers.expat import ExpatError
 
+import pytz
 from dict2xml import dict2xml
 from django.conf import settings
-from django.core.exceptions import (
-    ValidationError, PermissionDenied, MultipleObjectsReturned)
+from django.contrib.auth.models import User
+from django.core.exceptions import (MultipleObjectsReturned, PermissionDenied,
+                                    ValidationError)
 from django.core.files.storage import get_storage_class
 from django.core.mail import mail_admins
-from django.contrib.auth.models import User
 from django.db import IntegrityError, transaction
 from django.db.models import Q
-from django.http import HttpResponse
-from django.http import HttpResponseNotFound
-from django.http import StreamingHttpResponse
-from django.http import UnreadablePostError
+from django.http import (HttpResponse, HttpResponseNotFound,
+                         StreamingHttpResponse, UnreadablePostError)
 from django.shortcuts import get_object_or_404
+from django.utils import timezone
 from django.utils.encoding import DjangoUnicodeDecodeError
 from django.utils.translation import ugettext as _
-from django.utils import timezone
 from modilabs.utils.subprocess_timeout import ProcessTimedOut
-from pyxform.errors import PyXFormError
-from pyxform.xform2json import create_survey_element_from_xml
-from wsgiref.util import FileWrapper
+from raven.contrib.django.raven_compat.models import client
 
-from onadata.apps.logger.models import Attachment
-from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Attachment, Instance, XForm
 from onadata.apps.logger.models.instance import (
-    FormInactiveError,
-    InstanceHistory,
-    get_id_string_from_xml_str)
-from onadata.apps.logger.models import XForm
+    FormInactiveError, InstanceHistory, get_id_string_from_xml_str)
 from onadata.apps.logger.models.xform import XLSFormError
 from onadata.apps.logger.xform_instance_parser import (
-    InstanceEmptyError,
-    InstanceInvalidUserError,
-    InstanceMultipleNodeError,
-    NonUniqueFormIdError,
-    DuplicateInstance,
-    clean_and_parse_xml,
-    get_uuid_from_xml,
-    get_deprecated_uuid_from_xml,
-    get_submission_date_from_xml)
```

Comment:
This is the work of [yapf](https://github.com/google/yapf), I will add a config for it.